example-config.yaml

```diff
@@ -95,6 +95,10 @@ meta:
     # HTTP endpoint to request new proxy address from, for dynamically assigned proxies.
     # The endpoint must return a JSON body with a string field called proxy_url.
     get_proxy_from:
+    # Minimum interval between full reconnects in seconds, default is 1 hour
+    min_full_reconnect_interval_seconds: 3600
```

Comment:
I'm exposing MinFullReconnectIntervalSeconds in the config file, as it also affect the interval for fore refreshes.